osquery/tables/system/windows/security_profile_info_utils.h

```diff
@@ -0,0 +1,253 @@
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+
+#pragma once
+
+#include <osquery/core/tables.h>
+
+namespace osquery {
+namespace tables {
+
+/**
+ * @brief Manages access to SCE RPC client API exported in scecli.dll
+ */
+class SceClientHelper : public boost::noncopyable {
+  /**
+   * @brief SCE flag that indicates SCE SYSTEM profile request
+   */
+  static constexpr DWORD kSceSystemFlag = 300;
```

Comment:
nit: considering these are now class member variables, I think we can drop the `k` prefix for them, as that tends to be something we use for more globally scoped variables.

Reply:
Thanks Nick for spending time reviewing this! I really appreciate your time and effort. As for the nit changes on the member variables' names, I will address them today. Thanks!